CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing to Cyrus
+
+So, you want to contribute to Cyrus?  Great!
+
+You'll probably want to join the [cyrus-devel mailing
+list](https://www.cyrusimap.org/imap/support/feedback-mailing-lists.html#feedback-mailing-lists)
+where development issues get discussed.  You don't need to, but if you're
+considering doing a substantial amount of work, it's a good idea to ask about
+it first.
+
+This document is meant to be a quick overview of the most important things you
+need to know to get your work reviewed, approved, and into Cyrus.
+
+## Your Code
+
+Cyrus doesn't have a hard and fast style guide, *but it will*.  For now,
+consult the [Cyrus hacking
+docs](https://www.cyrusimap.org/imap/developer/guidance/hacking.html), which
+spell out some of the standards of formatting and construction.
+
+## The Tests
+
+You should run the tests.  Submitting a change that breaks existing tests isn't
+good for anybody!  Adding new tests to assert that your changes work is even
+better, and should be the goal.
+
+There are two kinds of tests:
+
+* The [Cassandane test
+  suite](https://www.cyrusimap.org/imap/developer/developer-testing.html) is an
+  integration test suite.  It can and should be run against your build of
+  Cyrus.  You need to get it from a separate repository.  (Follow that link!)
+* The [cunit tests](https://www.cyrusimap.org/imap/developer/unit-tests.html)
+  are located in the Cyrus IMAP repository, in `./cunit` and run by `make
+  check`.  You should run these, too.
+
+## Submitting Your Work
+
+We use GitHub, including pull requests.  Submit a pull request.  One of the
+committers should review it soon.  If they don't, the best place to ask for
```

Comment:
„Soon“ does not reflect what happens in practice.

Reply:
Agreed, but the same goes for other things in here.  First we right down what we're shooting for, then we'll keep working on it.

Reply:
From when on will review happen soon?